src/dispatch_wgl.c

```diff
@@ -27,8 +27,8 @@
 
 #include "dispatch_common.h"
 
-static bool first_context_current = false;
-static bool already_switched_to_dispatch_table = false;
+//static bool first_context_current = false;
+// static bool already_switched_to_dispatch_table = false;
```

Comment:
Please, don't leave commented out lines.

Reply:
Sorry for the dirty code. I'll try to clean them up.

Reply:
Removed.